components/stateTracker.js

```diff
@@ -90,6 +90,7 @@ StateTracker.prototype.bind = function() {
     var self = this;
     $(window).on('popstate', function() {
         if (!self.disabled) {
+            _.extend(history.state || {}, {_stateFromHistory: true});
```

Comment:
Why *_*? This method is interfacing enough. *isStateFromHistory* name will better imho.